lib/src/signed_video_tlv.h

```diff
@@ -149,4 +149,24 @@ tlv_find_and_decode_recurrent_tags(signed_video_t *self,
     const uint8_t *tlv_data,
     size_t tlv_data_size);
 
+/**
+ * @brief Helper to get only the optional tags as an array.
+ *
+ * @param num_of_man_tags Number of optional tags.
```

Comment:
Rephrase to "A pointer to a location where the number of optional tags will be written."